1.20/README.md

```diff
@@ -102,7 +102,7 @@ This step usually consists of at least these parts:
 * moving configuration files to the correct place (if available in the application source code)
 * setting the default command in the resulting image
 
-For all these three parts, you can either set up all manually and use the `nginx` command explicitly in the Dockerfile ([3.1.](#31-to-use-own-setup-create-a-dockerfile-with-this-content)), or you can use the Source-to-Image scripts inside the image ([3.2.](#32-to-use-the-source-to-image-scripts-and-build-an-image-using-a-dockerfile-create-a-dockerfile-with-this-content). For more information about these scripts, which enable you to set-up and run the nginx daemon, see the "Source-to-Image framework and scripts" section above.
+For all these three parts, you can either set up all manually and use the `nginx` command explicitly in the Dockerfile ([3.1.](#31-to-use-own-setup-create-a-dockerfile-with-this-content)), or you can use the Source-to-Image scripts inside the image ([3.2.](#32-to-use-the-source-to-image-scripts-and-build-an-image-using-a-dockerfile-create-a-dockerfile-with-this-content), that already know how to set-up and run some common Nginx applications.
```

Comment:
Why was the reference to the "Source-to-Image framework and scripts" section removed?

Reply:
The same documentation style like here: https://github.com/sclorg/s2i-perl-container/tree/master/5.30#3-prepare-an-application-inside-a-container or here: https://github.com/sclorg/s2i-nodejs-container/tree/master/14#3-prepare-an-application-inside-a-container . But some part is still missing. I will fix it like in `s2i-perl-container` .